### DISCUSSION TITLE
Cyberduck PDF Loading Error

### DISCUSSION THREAD

**Discussion by acarmichael624** [iterate-ch/cyberduck#16625] [Q&A]
I am trying to upload a PDF file through CyberDuck content for our site in BigCommerce.
I keep getting the same error message: “Cannot retry request with a non-repeatable request entity.” It says my file uploaded and then failed to upload. (File hasn’t uploaded).

I have worked with BigCommerce support as well as our internal IT team and have not had any success in loading the file. We’ve tried updating to latest version and reverting to older versions with no luck. Do you have any advice?

**Comment by dkocher** [comment]
This was previously discussed in #16308 and we track it as a bug in [^1]. In general the error is masquerading the underlying cause which is either an authentication or interoperability failure. 

Refer to [^2] for documentation by BigCommerce. It suggests that filenames should contain no whitespace.

[^1]: #16024
[^2]: https://support.bigcommerce.com/s/article/How-do-I-add-and-link-to-a-file-in-my-store?language=en_US#upload-a-file
